spec/models/local_authority/postcode_spec.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: local_authority_postcodes
+#
+#  gss_code   :string           not null
+#  value      :string           not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+# Indexes
+#
+#  index_local_authority_postcodes_on_gss_code  (gss_code)
+#  index_local_authority_postcodes_on_value     (value) UNIQUE
+#
+describe LocalAuthority::Postcode, type: :model do
+  it { should normalize(:value).from(" Sw111AA ").to("SW11 1AA") }
+  it { should normalize(:value).from(" Sw11.   1aA ").to("SW11 1AA") }
+
+  describe "normalize_postcode" do
```

Comment:
Do we need to test this function on its own, it feels like to me like it should be private (or even just directly in the block) and we can test it using `should normalize(:value)` like the ones above?

Reply:
Yeah, I originally made it a standalone method so that it could be called from the `LocalAuthority.for_postcode(..)` method - but now that external call is not needed (thanks to the merged scope) it can go back inside the block